sdk/sdk.go

```diff
@@ -90,9 +90,9 @@ func (sdk *ShannonSDK) GetSessionSupplierEndpoints(
 	return sessionSuppliers, nil
 }
 
-// GetApplicationsDelegatingToGateway returns the application addresses that are
-// delegating to the given gateway address.
-func (sdk *ShannonSDK) GetApplicationsDelegatingToGateway(
+// GetGatewayDelegatingApplications returns the applications that are delegating
```

Comment:
Why did you revert this back from the changes that I made?

I renamed it in  my commit.

Reply:
Probably a bad merge. Will put it back.